ingest/ingest-datasets-old.py

```diff
@@ -0,0 +1,392 @@
+__author__ = 'szednik'
+# as template, altered by Hao to fit the dataset ingest process
+# archived as developing a new ingest script utilizing a sepatate .py file for common functions
+
+from SPARQLWrapper import SPARQLWrapper, JSON
+from rdflib import Namespace, RDF
+import json
+import requests
+import multiprocessing
+import itertools
+from itertools import chain
+import functools
+import argparse
+import warnings
+import pprint
+
+
+class Maybe:
```

Comment:
Yep. Its the -old.py. Just to archive the old version in case we have some problems in the development..